source/slang/slang-ir-insts.lua

```diff
@@ -496,11 +496,11 @@ local insts = {
 			{
 				ptr_constant = { struct_name = "PtrLit" },
 			},
+			{ void_constant = { struct_name = "VoidLit" } },
```

Comment:
this is reordered to make checking for a 64-bit payload constant possible with a range comparison rather than checking each case (by making them consecutive)